client/js/components/user/orgaDataEntry/OrganisationCompetence.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <fieldset
+    v-if="hasPermission('field_organisation_competence')"
+    id="organisationCompetence"
+    class="w-3/4">
+    <legend class="font-size-large weight--normal u-mb-0_75">
+      {{ Translator.trans('competence') }}
+    </legend>
+    <div
+      class="w-full mb-3">
```

Comment:
Did you run `yarn lint`? I don't know if it's the github interface, but it seems there are a couple of formatting issues in this file

Reply:
Can't figure out how to run it on just one file atm, but it looks fine to me (I changed only the div) here [785572d](https://github.com/demos-europe/demosplan-core/pull/4237/commits/785572dcbbcbd10c664361ed3411ab48bbd8de84)